Gridy/Model/Position.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Position.swift
+//  Gridy
+//
+//  Created by Alexandra Ivanova on 11/10/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+/// Calculate the frame of squares inside a view
+class Position {
+  private let safeArea: UIEdgeInsets
+  private let height: CGFloat
+  private let width: CGFloat
+  private let isLandscape: Bool
+  private var gapLength: CGFloat = 0
+  private let allGaps: CGFloat
+  private var countInRow: CGFloat = 0
+  private var length: CGFloat = 0
+  private let allSquares: CGFloat
+  private let offset: CGFloat
+  private var marginX: CGFloat = 0
+  private var marginY: CGFloat = 0
+  private var numberSquares: Int = 0
```

Comment:
Many of the properties here seem unused or redundant with other properties

Reply:
I'll cleanup this class later